examples/nextjs/README.md

```diff
@@ -0,0 +1,43 @@
+# Next.js Contentful live preview SDK example
+
+This is an example project that demonstrates how to use the `@contentful/live-preview` SDK with a Next.js application. The SDK provides live preview functionality for content changes and the inspector mode for your Contentful space.
+
+## 1. Installation
+
+Install the dependencies:
+
+```bash
+npm install
```

Comment:
npm or yarn?

Reply:
We have a package-lock file so it's recommended to use npm :)

Reply:
Ahh ok, I missed that. Was a bit confused since we use yarn for the main package